specification/cosmos-db/resource-manager/readme.md

```diff
@@ -27,21 +27,33 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-2020-09
+tag: package-2021-01
 ```
 
 
+### Tag: package-2021-01
+
+These settings apply only when `--tag=package-2021-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-01'
```

Comment:
You should also make changes in other Readme files in this folder.

Reply:
done